In file included from ../../components/payments/content/payment_method_manifest_table.cc:5:
../../components/payments/content/payment_method_manifest_table.h:153:8: error: no template named 'optional' in namespace 'std'
  153 |   std::optional<std::vector<uint8_t>> GetBrowserBoundKey(
      |   ~~~~~^
1 error generated.


--- a/components/payments/content/payment_method_manifest_table.h
+++ b/components/payments/content/payment_method_manifest_table.h
@@ -6,6 +6,7 @@
 #define COMPONENTS_PAYMENTS_CONTENT_PAYMENT_METHOD_MANIFEST_TABLE_H_
 
 #include <memory>
+#include <optional>
 #include <string>
 #include <vector>
 
